src/api/resources.ts

```diff
@@ -44,3 +48,15 @@ export async function get(id: string) {
 
   return result
 }
+
+export async function createComment(
+  resourceId: number,
+  params: IResourceCommentParams,
+) {
+  const result = await apiClient
+    .post(`resources/${resourceId}/comments`, params)
+    .then((r) => r.data)
+    .catch(() => false)
```

Comment:
Might help if you log the error here while you're debugging (but dont commit it this way because console.log wont pass lint check):
```javascript
    .catch((e) =>console.log(e) && false)
```

Alternatively, you can look at the failed request response in the network tab of the browser tools.